### PR TITLE
Adding in data connect too

### PR DESCRIPTION
**Pull request by joehan**
### Description
Some cleanup + do it for data connect too!

### CODE REVIEW

File: src/requireTosAcceptance.ts [hunk 1]
```diff
@@ -30,6 +37,6 @@ async function requireTos(tosId: TosId): Promise<void> {
   }
   const console = consoleLandingPage.get(tosId) || consoleOrigin();
   throw new FirebaseError(
-    `Your account has not accepted the required Terms of Service for this action. Please accept the Terms of Service and try again. ${console}`,
+    `Your account is missing the required terms of service for this action. Please accept the Terms of Service and try again. ${console}`,
```
Comment (tonyjhuang) [inline src/requireTosAcceptance.ts:40]: revert?

File: src/test/requireTosAcceptance.spec.ts [hunk 2]
```diff
@@ -68,7 +70,9 @@ describe("requireTosAcceptance", () => {
       .reply(200, SAMPLE_RESPONSE);
 
     await expect(requireTosAcceptance(APPHOSTING_TOS_ID)(SAMPLE_OPTIONS)).to.be.rejectedWith(
-      "Terms of Service",
+      "Your account is missing the required terms of service for this action. Please accept the Terms of Service and try again. https://console.firebase.google.com/project/_/apphosting",
```
Comment (tonyjhuang) [inline src/test/requireTosAcceptance.spec.ts:73]: revert?

**Review by tonyjhuang — COMMENTED** [review]
looks like you have some stale changes

File: src/requireTosAcceptance.ts (hunk 1 above)
Comment (joehan) [reply src/requireTosAcceptance.ts:40]: Will do, brain farted during rebase

**Review by tonyjhuang — APPROVED** [review]

**Pull request merged**